sql/moz-fx-data-shared-prod/static/us_zip_code_prefixes_v1/description.txt

```diff
@@ -0,0 +1,3 @@
+Mappings from three-digit US ZIP code prefixes to US state codes.
+
+Data source: https://simple.wikipedia.org/wiki/List_of_ZIP_Code_prefixes
```

Comment:
This was the first time I've come across [Simple English Wikipedia](https://simple.wikipedia.org/wiki/Simple_English_Wikipedia).  I cited the Simple English Wikipedia version because it was the easiest place to quickly get the data in a reasonable format, compared to the [equivalent page on normal Wikipedia](https://en.wikipedia.org/wiki/List_of_ZIP_Code_prefixes) which has a highly formatted table that'd be much harder to copy the data from.